recipes/mocknetworkaccessmanager/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,12 @@
+#include "MockNetworkAccessManager.hpp"
+
+int main(void) {
+    MockNetworkAccess::Manager< QNetworkAccessManager > mnam;
+
+    mnam.whenGet( QUrl( "http://example.com/hello" ) )
+        .has( MockNetworkAccess::Predicates::HeaderMatching( QNetworkRequest::UserAgentHeader,
+                    QRegularExpression( ".*MyNetworkClient/.*" ) ) )
+        .reply().withBody( QJsonDocument::fromJson( "{\"hello\":\"world\"}" ) );
```

Comment:
```suggestion
    mnam.whenGet( QUrl( "http://example.com/hello" ) )
        .has( MockNetworkAccess::Predicates::HeaderMatching( QNetworkRequest::UserAgentHeader,
                    QRegularExpression( ".*MyNetworkClient/.*" ) ) )
        .reply().withBody( QJsonDocument::fromJson( "{\"hello\":\"world\"}" ) );
```
Please, do not use an example using networking. The CI is very restrictive when using it and may fail. Plus, it may block few users that are managed by enterprise firewalls etc.

Reply:
This library is mocking the QNetworkAccessManager so it doesn't do any request at all.

Reply:
Ah, good catch! Thank you!!!